rfcs/text/0000-rfc-application-layer-bundle-types-and-definitions.md

```diff
@@ -0,0 +1,424 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Application Layer Bundle Types and Definitions
+
+
+## Summary
+
+This RFC is offered as a refinement and generization of the [RFC: HCA DCP Bundle Types and Definitions]( https://github.com/HumanCellAtlas/dcp-community/pull/93). 
+
+This RFC describes:
+
+1. An application/metadata level mechanism to define bundle types and a simple bundle type hierarchy.
+
+1. An application/metadata level method to specify the type of a bundle.
+ 
+1. Several bundle types designed to match current usage and near term expected usage. 
+
+### Key Differences from RFC: HCA DCP Bundle Types and Definitions
+
+This proposal differs from [RFC: HCA DCP Bundle Types and Definitions](https://github.com/HumanCellAtlas/dcp-community/pull/93/files) mainly in that:
+
+1. Type information is added to a type.json  metadata file rather than added to the DSS bundle.json file.
+
+1. Type information is expressed in JSON rather than of [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) media type syntax.
+
+1. The schema and bundle types are represented as JSON schema in the metadata schema repo and documented on the Data Portal rather than maintained in a DSS registry.
```

Comment:
As mention above, I don't think it is necessary to have a separate file so that we can have a schema.  Also, the current schema metadata repo is experimental metadata, so we need to somehow separate them, although I don't think putting them into different places serves the needs of the user, only distinguishing how bundle metadata and scientific metadata are not the same things.

Reply:
I have added "What repo should the schema go in?" to the unresolved questions.